src/components/LoanCards/Buttons/LendCtaExp.vue

```diff
@@ -194,6 +202,14 @@ export default {
 			return this.loan?.minNoteSize ?? '';
 		},
 		unreservedAmount() {
+			if (this.enableRelendingExp) {
+				if (this.enableFiveDollarsNotes) {
+					if (this.userBalance > 20) return '25';
+					return '5';
+				}
+				if (this.loan.unreservedAmount > 25) return '25';
+				return this.loan?.unreservedAmount ?? '';
```

Comment:
I think it's dangerous to change the meaning on `unreservedAmount` here. We have a decent amount of logic around what "unreserved" means. I think it would be safer to instead change the `amountLeft` passed to the `LendAmountButton` based on whether the experiment is applied.